cmd/scan.go

```diff
@@ -43,15 +48,32 @@ var scanCmd = &cobra.Command{
 	Long: `
 Scans a Go module directory. To scan the current directory recursively, use gokart scan. To scan a specific directory, use gokart scan <directory>.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) == 0 {
-			// recursively scan the current directory if no arguments are passed in
-			args = append(args, "./...")
-		}
 		sarif, _ := cmd.Flags().GetBool("sarif")
 		globals, _ := cmd.Flags().GetBool("globalsTainted")
 		verbose, _ := cmd.Flags().GetBool("verbose")
 		debug, _ := cmd.Flags().GetBool("debug")
 		util.InitConfig(globals, sarif, verbose, debug, outputPath, yml)
+		
+		// If gomodname flag is set to a non-empty value then clone the repo and scan it
+		if len(goModName) != 0 {
+			modDirName, err := util.ParseModuleName(goModName)
+			if err != nil {
+				log.Fatal(err)
+			}
+			err = util.CloneModule(modDirName, "https://"+goModName)
+			if err != nil {
+				log.Fatal("GoKart was unable to get the new racetrack. Ensure track repository is open to the public or that your access tokens are configured correctly for Private ones.")
```

Comment:
It said _your access tokens are configured correctly for Private ones_, but how do I define those things?